### ISSUE TITLE
Write Bio - Site Builder: Henry Latourrette

### ISSUE THREAD

**Issue by henlatourrette** [100Automations/Website#235]
### Overview
We need some info to put on our site about you, so that we highlight your contributions.

### Action item
- [ ] Provide 
   - [x] Name
   - [x] Bio: _A 1-2 paragraph professional bio_
   - [x] Quote:  _A quote from you about automations_
   - [x] Motivations
      - [x] Internal:  _What are your internal motivations for creating the 100 Automations site?_
      - [x] External: _What are your external motivations for creating the 100 Automations site?_
   - [x] Picture - _Can we use your LinkedIn pic? If yes, please provide your LinkedIn URL_

### Resources
[https://github.com/100Automations/Website/issues/120](https://github.com/100Automations/Website/issues/120)
[https://github.com/100Automations/Website/issues/66](https://github.com/100Automations/Website/issues/66)
[https://github.com/100Automations/Website/issues/81](https://github.com/100Automations/Website/issues/81)

**Comment by henlatourrette** [comment]
**Name**: Henry Latourrette

**Bio**: Hi! I'm a member of the Product Management team here at 100 Automations, and the International Roaming Manager at a TelCo here in Paraguay. I have worked as a Software Engineer and as a GSM Network engineer for 8+ years. A few years ago I've also been a Sales Rep - a fun way to meet new people.

In the early 2000's I went to college for an Electronics Engineering degree but ended up getting a Marketing one in 2014. Still, always involved with the tech community and proud of being part of such an impactful Open Source Project like this.

Besides work, I enjoy playing football (soccer) and going to live concerts. On vacations, I usually go to the beach and I love discovering new places.

**Quote**: 
Instead of reinventing the wheel, let's focus our time building upon it.

**Motivations**:

**Internal**: 
I want to learn from new technologies and new ways to collaborate with remote and Open Source teams. 

**External**:
I feel this is a way to give back to the community and to have an impact on the Open Source projects of the future.

**Picture**:
Check my [LinkedIn profile Pic](https://www.linkedin.com/in/henry-latourrette-8419571a/)
